src/main/java/org/opentripplanner/model/TripPattern.java

```diff
@@ -112,7 +111,7 @@ public TransitMode getMode() {
         return route.getMode();
     }
 
-    public final String getNetexSubmode() {
+    public String getNetexSubmode() {
```

Comment:
These don't belong to this PR?

Reply:
Intellij showed a warning (final method in final class is meaningless) and I tend to do some clean up when I edit files. Do these cleanup confuse more than they help?